docs/packaging/packaging-with-visual-studio.md

```diff
@@ -0,0 +1,81 @@
+Click "Create" and then select "10.0.26100.0" (or later) for Target Platform Version (TPV) 
+and Target Platform Minimum Version (TPMinV). 
+
+![image](images/vs/02-Blank-App-Packaged-With-Windows-Application-Packaging-Project.png)
+![image](images/vs/03-New-Project.png)
+![image](images/vs/04-New-Windows-Project.png)
+
+## Step 3 - Set NuGet to use Microsoft.Windows.SDK.BuildTools version 10.0.26100.1 or later
+Go to Project -> "Manage NuGet Packages" to install Microsoft.Windows.SDK.BuildTools 
+version 10.0.26100.1 (or later).
+
+![image](images/vs/05-Manage-NuGet-Packages.png)
+![image](images/vs/06-Microsoft-Windows-SDK-BuildTools.png)
+![image](images/vs/07-SDK-BuildTools-Info.png)
+
+## Step 4 - Edit the Packaging.appxmanifest file
+In the manifest file, the following changes will need to be made:
+
+**Note**: Isolated win32 applications are not compatible with other application types within the same package.
+
+* Add `xmlns:previewsecurity2="http://schemas.microsoft.com/appx/manifest/preview/windows10/security/2"`
+to the `<Package>` element if it's not there already.
+
+	* Add `previewsecurity2` to `IgnorableNamespaces` at the end of the `<Package>` element.
+
+* Add `xmlns:uap10="http://schemas.microsoft.com/appx/manifest/uap/windows10/10"` to the `<Package>` element
+if it's not there already.
+
+	* Add `uap10` to `IgnorableNamespaces` at the end of the `<Package>` element.
+
+* In `<Dependencies>` change `TargetDeviceFamily` to
+`<TargetDeviceFamily Name="Windows.Desktop" MinVersion="10.0.25357.0" MaxVersionTested="10.0.25357.0" />`.
+
+	* **Note**: Not all features are available in the minimum build, check out the [release notes](../../relnotes/windows-release-notes.md) for more detailed information.
+
+* In `<Application>` replace any existing entrypoint/trustlevel/runtimebehavior with
+`uap10:TrustLevel="appContainer" previewsecurity2:RuntimeBehavior="appSilo"`.
+
+* In `<Application>` extensions, remove any `EntryPoints=*` or `Executable=*` as those are inherited from the parent `<Application>`
+
+* Add `desktop7:Scope="user"` to the extension element for `windows.protocol`.
+
+* **Note**: By default, Visual Studio will automatically add `<rescap:Capability name="runFullTrust">` to
+`<Capabilities>` due to the app being a packaged Win32. This should be removed unless
+the app has other manifested extensions which can affect the user global state, such as
+`comServer` or `FirewallRules`, since those require the `runFullTrust` capability.
+
+![image](images/vs/09-PackagingAppx-File.png)
+
+## Step 5 - Build and Publish App Packages
+Build the Visual Studio solution. 
+
+![image](images/vs/10-Build-Solution.png)
+
+Publish the App Package by using the "Create App Packages..." wizard.
+
+![image](images/vs/11-Create-App-Packages.png)
+![image](images/vs/12-Create-App-Packages-Create.png)
+![image](images/vs/13-Create-App-Packages-Success.png)
+
+## Step 6 - Install App
+Use Run the Install.ps1 with PowerShell to install the App.
```

Comment:
is "Use Run the Install.ps1" the correct grammar? This is where *Italic terms` helps. I don't know if I should be looking for a button named *Run the Install.ps1*. Also why do you think the installation is part of packaging? That always feels a bit weird to me.

Reply:
I prefer tutorials with end-to-end instructions, but I am okay to omit step 6 if you feel it's unnecessary.

Reply:
From my perspective, this is a packaging documentation. The user should get a package (msix) in the end. The MPT docs give an MSIX file at the end of the documentation. Rather than how to install the package, I think we should guide users to find the MSIX package the build in the previous step.